nomad/core_sched_test.go

```diff
@@ -209,21 +317,27 @@ func TestCoreScheduler_EvalGC_BatchStopped(t *testing.T) {
 	job := mock.Job()
 	job.Type = structs.JobTypeBatch
 	job.Status = structs.JobStatusDead
+	job.Stop = true
+	job.TaskGroups[0].ReschedulePolicy = &structs.ReschedulePolicy{
+		Attempts: 0,
+		Interval: 0 * time.Second,
```

Comment:
Can this just be 0?

Reply:
0 will work. I added the time.Minute for visual clarity so I know when quickly scanning the test that interval takes a duration.